prov/rxm/src/rxm_init.c

```diff
@@ -66,33 +63,17 @@ int rxm_info_to_core(uint32_t version, const struct fi_info *hints,
 		if (hints->caps & (FI_MSG | FI_TAGGED))
 			core_info->caps |= FI_RMA;
 
-		/* No fi_info modes apart from those handled earlier in
-		 * this function can be passed along to the core provider */
-
-		if (hints->domain_attr) {
-			if (FI_VERSION_GE(version, FI_VERSION(1, 5))) {
-				/* Allow only those mr modes that can be
-				 * passed along to the core provider */
-				core_info->domain_attr->mr_mode |=
-					hints->domain_attr->mr_mode &
-					OFI_MR_BASIC_MAP;
```

Comment:
RxM would not handle a provider requiring modes like FI_MR_ENDPOINT, FI_MR_RAW, etc when it uses RMA for large message protocol. So we can't pass those bits to the core provider.